datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -358,6 +360,7 @@ public class GmsGraphQLEngine {
 
     public GmsGraphQLEngine(
         final EntityClient entityClient,
+        final EntityClient restliEntityClient,
```

Comment:
If we can avoid doing this it will be very useful.

At minimum, we should explicitly type this as RestliEntityClient to avoid accidentally calling the wrong client... 

If there's any way to maintain functional parity between the 2 clients it will be greatly preferred!